conu/backend/docker/client.py

```diff
@@ -3,13 +3,19 @@
 """
 from __future__ import print_function, unicode_literals
 
+from conu.utils import check_docker_command_works
+
 import docker
 
+
 client = None
 
 
 def get_client():
     global client
     if client is None:
         client = docker.APIClient(version="auto")
+        # FIXME: once we implement `run_via_api`, move this elsewhere; ideally to run_via_binary
```

Comment:
agreed, I am not aware of any other methods that should use docker binary instead of API.